assets/app/view/game/corporation.rb

```diff
@@ -38,9 +39,17 @@ def render
         card_style = {
           cursor: 'pointer',
         }
-        card_style[:border] = '4px solid' if @game.round.can_act?(@corporation)
+
         card_style[:display] = @display
 
+        unless interactive?
+          factor = Native(`window.matchMedia('(prefers-color-scheme: dark)').matches`) ? 0.8 : 0.5
```

Comment:
I think this should be avoided. I prefer dark mode and have it enabled on my computer, but on my phone I don't have that setting. In both cases I use the same color scheme (defined on the profile settings).

I'd suggest you just pick an intermediate value like 0.6

Reply:
This is used elsewhere in the system, however I could calculate it based on whether the background color is light or dark rather than using a window function.

Reply:
I've changed this to use math based on bg2 instead of the dark mode detection.

<img width="483" alt="Screen Shot 2021-01-01 at 8 24 30 AM" src="https://user-images.githubusercontent.com/15675400/103441411-0e1c0280-4c0b-11eb-8359-96c27f532fd0.png">
<img width="490" alt="Screen Shot 2021-01-01 at 8 24 53 AM" src="https://user-images.githubusercontent.com/15675400/103441412-0eb49900-4c0b-11eb-9f9e-f7bb9ea478e9.png">

Reply:
Afaict it's only used twice. One of them is to set the default. The other is in the spreadsheet. These 2 screenshots are taken with the same account, but on different browsers:

![image](https://user-images.githubusercontent.com/3337865/103441949-87d2e100-4c49-11eb-8bdc-ab4ff9dc59c1.png)

![image](https://user-images.githubusercontent.com/3337865/103441954-97eac080-4c49-11eb-8b11-6170e598e7e2.png)

I'll try to open a PR changing the spreadsheet to work like what you did.